catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/pojo/FormTemplate.java

```diff
@@ -30,12 +32,25 @@
   @JsonProperty("filterTemplate")
   private FilterNode root;
 
-  public FormTemplate(Metacard metacard, FilterNode root) {
+  @JsonProperty("accessIndividuals")
+  private List<Serializable> accessIndividuals;
+
+  @JsonProperty("accessGroups")
+  private List<Serializable> accessGroups;
+
+  public FormTemplate(
+      Metacard metacard,
+      FilterNode root,
+      List<Serializable> accessIndividuals,
+      List<Serializable> accessGroups) {
     super(metacard);
     this.root = root;
+    this.accessIndividuals = accessIndividuals;
+    this.accessGroups = accessGroups;
```

Comment:
I wonder if there's a way we can simplify this. If we need to display sharing comprehensively on a group & individual level then it might be unavoidable.

Reply:
This seemed like one of the cleanest ways to play nice with Intrigue.